x-pack/plugin/ilm/qa/multi-node/src/test/java/org/elasticsearch/xpack/indexlifecycle/TimeSeriesLifecycleActionsIT.java

```diff
@@ -393,9 +393,6 @@ public void testDeleteDuringSnapshot() throws Exception {
         // assert that snapshot is still in progress and clean up
         assertThat(getSnapshotState("snapshot"), equalTo("SUCCESS"));
         assertOK(client().performRequest(new Request("DELETE", "/_snapshot/repo/snapshot")));
-        ResponseException e = expectThrows(ResponseException.class,
```

Comment:
What do we get here now, shouldn't we still see a 404 if we simply check a single repo?

Reply:
No, we will get 200 OK. And this is consistent with how other batch API currently work. For example, multi-get API. If you get even a single item that does not exist, you will get 200 OK.

Reply:
It's still possible to replace the old assertion with a new check that digs into received JSON, I'm just not sure if it worths it, considering that we need to deal with raw JSON.

Reply:
Makes sense :)